Source/Events.Store.MongoDB/Streams/StreamFetcher.cs

```diff
@@ -111,6 +111,20 @@ public async Task<Try<IEnumerable<StreamEvent>>> Fetch(StreamPosition streamPosi
         }
     }
 
+    /// <inheritdoc />
+    public async Task<Try<StreamPosition>> GetNextStreamPosition(CancellationToken cancellationToken)
+    {
+        try
+        {
+            var numEvents = await _collection.CountDocumentsAsync(_filter.Empty, cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
I don't know, but it might be possible that getting the last document sorted ascending is faster? But we could just leave this for optimisation later...

Reply:
This will eventually at some point all go away so I wouldn't bother doing this